etc/config/c++.amazon.properties

```diff
@@ -881,6 +881,8 @@ libs.glm.versions.0995.version=0.9.9.5
 libs.glm.versions.0995.path=/opt/compiler-explorer/libs/glm/0.9.9.5
 libs.glm.versions.0996.version=0.9.9.6
 libs.glm.versions.0996.path=/opt/compiler-explorer/libs/glm/0.9.9.6
+libs.glm.versions.0997.version=0.9.9.7
```

Comment:
I think this one is missing the `0997` in the `libs.glm.versions` - will add that myself (thanks so much for these)

Reply:
You're right. I knew I would mess one of them up :D